xtask/src/ci.rs

```diff
@@ -10,7 +10,7 @@ mod spec_links;
 use spec_links::check_spec_links;
 
 const MSRV: &str = "1.55";
-const NIGHTLY: &str = "nightly-2022-03-23";
```

Comment:
Why are we going away from pinning nightly to going to the latest nightly? It could maybe break CI, or create weird environments.

We should probably just update it, not go to rolling nightly.

Reply:
That's what we've been doing previously too. I don't really want to be in the business of having to update this manually. We only had it pinned recently because something broke temporarily.

Reply:
Seems reasonable then, but I'm just noting that it might introduce CI weirdness and the likes.

Reply:
Yeah, I know but I consider it better than the alternatives. Do you wanna approve the PR then? 🙂